docs/DeveloperGuide.md

```diff
@@ -36,3 +33,115 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+
+<!-- Zhan hong working on this section-->
+# Review Feature
+**API:** [Review.java](https://github.com/AY2223S2-CS2113-W12-4/tp/blob/master/src/main/java/Review.java)
+
+Here's a partial class diagram of the `Review` component:
+
+![](/PUMLFiles/Review/review1.png)
+
+The `Review` component contains both a descriptive review `reviewText`,
+and a numerical review `reviewStars` (0 to 5 inclusive). Both are modified through the
+`setReview()` method.
+
+`Review` implements [`Comparable`](https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html), and is compared
+through simply comparing `reviewStars`.
+
+How the `Review` component works:
+
+**Step 1.** `setReview()` is called, which creates a [`Scanner`](https://docs.oracle.com/javase/7/docs/api/java/util/Scanner.html)
```

Comment:
This Sequence diagram is located in between step 1 and step 2. I am not sure if this is intended but I feel that it would be better to place it before step 1 so that I can follow through all the steps much more easily.